backend/.env.example

```
@@ -1,39 +1,74 @@
-COOKIE_SECRET=your_cookie_secret
-CSRF_SECRET=your_csrf_secret_min_32_chars
+PORT=3000
+REDIS_URL=rediss://user:password@host:6379
+COOKIE_SECRET=your_cookie_secret_min_32_chars
+BACKUP_ENCRYPTION_KEY=your_backup_encryption_key_64_chars
+ENCRYPTION_KEY=your_encryption_key_64_chars
+VERCEL_API_TOKEN=your_vercel_api_token
+MOCK_AI=false
+SRA_API_KEY=sra_live_your_sra_api_key
+ENCRYPTION_SALT=sra-encryption-salt-v1
+BACKUP_ENCRYPTION_SALT=salt
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BACKUP_ENCRYPTION_SALT key should go before the COOKIE_SECRET key

(UnorderedKey)